tasks/promote.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Check if image suffix is provided
+  fail:
+    msg: "os_images_name_suffix is empty please provide it."
+  when: os_images_name_suffix is defined and os_images_name_suffix|length == 0
```

Comment:
when: os_images_name_suffix is None (might be shorter)

Reply:
Unfortunately this does not work as intended:
```fatal: [localhost]: FAILED! => {"msg": "The conditional check 'os_images_name_suffix is None' failed. The error was: No test named 'None' found.\n\nThe error appears to be in '/home/cloud-user/smslab-config/ansible/roles/stackhpc.os-images/tasks/promote.yml': line 2, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n---\n- name: Check if image suffix is provided\n  ^ here\n"}```

Reply:
oh well, then leave length, but add spaces - os_images_name_suffix | length == 0